containers/podman-desktop
# Image Checker UI

### What does this PR do?

This PR adds a UI for image checkers.

### Screenshot/screencast of this PR

https://github.com/containers/podman-desktop/assets/9973512/d409b774-b92d-45bc-815f-a4f575d45ebf

### What issues does this PR fix or reference?

Fixes #4707 

### How to test this PR?

You can install a fake image checker from the image at `quay.io/phmartin/fake-image-checker:v0.0.3` (sources: https://github.com/feloy/podman-desktop-extension-fake-image-checker)

Go to the details page of an image, and click the Check tab. Use can use the Abort button to ask the image checker to stop processing.
